frontend/src/components/ElectionForm/ElectionForm.tsx

```diff
@@ -130,26 +264,401 @@ const ElectionForm = ({ authSession, onSubmitElection, prevElectionData, submitT
+                        Option1={{
+                            Answer: 'Yes',
+                            HelpText: 'Voters will create an account and verify their email adress.',
+                            GotoPage: 'Scenario3',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        Option2={{
+                            Answer: 'No',
+                            HelpText: 'Voters will be given a unique link that will allow them to vote.',
+                            GotoPage: 'Scenario4',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        BackPage='Invitations?'
+                        setPage={setPage}
+                    />
+                    <Question
+                        Enable={page === 'Register?'}
+                        Question='Would you like to require voters to register with star.vote to vote?'
+                        HelpText=''
+                        Option1={{
+                            Answer: 'Yes',
+                            HelpText: 'Voters will create an account and verify their email adress.',
+                            GotoPage: 'Scenario8',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        Option2={{
+                            Answer: 'No',
+                            HelpText: '',
+                            GotoPage: 'Scenario9',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        BackPage='VoterList?'
+                        setPage={setPage}
+                    />
+
+                    <Question
+                        Enable={page === 'CustomRegister?'}
+                        Question='Would you like to add additional custom registration?'
+                        HelpText=''
+                        Option1={{
+                            Answer: 'Yes',
+                            HelpText: 'Voters will also provide additional info to cast a provisional ballot.',
+                            GotoPage: 'Scenario7',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        Option2={{
+                            Answer: 'No',
+                            HelpText: 'Voters will just create a star.vote account',
+                            GotoPage: 'Scenario8',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        BackPage='Register?'
+                        setPage={setPage}
+                    />
+                    <Question
+                        Enable={page === 'Scenario1'}
```

Comment:
nit

Ideally we could have more descriptive names for the configurations rather than ScenarioN but a concise name is probably tricky with so many parameters

Reply:
Yea I I kind of gave up on descriptive names by this point. I'll think about it for the future.